### DISCUSSION TITLE
Problema al ejecutar los test

### DISCUSSION THREAD

**Discussion by JuanCarlosRL** [Aparking/AparKing_Backend#92] [Show and tell]
**Descripción:**

Hemos identificado un problema recurrente al intentar ejecutar todas las pruebas unitarias de nuestra aplicación de manera conjunta. Este inconveniente fue mencionado previamente en la issue #13, pero persiste y afecta el flujo de trabajo.

**Detalle del problema:**

Al ejecutar el comando `python manage.py test` para correr todas las pruebas unitarias, el comando es incapaz de localizar y ejecutar los tests. Este comportamiento impide la ejecución automática de todas las pruebas, una práctica estándar y crucial para asegurar la calidad y funcionalidad de nuestra aplicación.

**Solución temporal:**

Por el momento, hemos encontrado que la única manera de ejecutar las pruebas es hacerlo individualmente, lo cual es ineficiente y consume una cantidad significativa de tiempo que podría ser mejor empleado en el desarrollo.

<img width="356" alt="image" src="https://github.com/Aparking/AparKing_Backend/assets/72872031/19b90e7a-ceb8-40ab-be3e-b2fdc807004d">

**Comment by virolialo** [comment]
Hola JC!

Está muy bien que hayas querido reflejar este error en GitHub, pero creo que es mejor dejarlo reflejado en Discussions, teniendo en cuenta que no es una tarea sino un mensaje de ayuda sobre el error sobre los tests.

**Comment by perealberto** [comment]
@JuanCarlosRL se terminó de solucionar este problema?
